scripts/test/SANS/command_interface/batch_csv_file_parser_test.py

```diff
@@ -5,6 +5,7 @@
 #   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
 # SPDX - License - Identifier: GPL - 3.0 +
 import os
+import tempfile
```

Comment:
Everything looks good , but tempfile is imported and i cant find an occurence where it is used.